server/game/player.js

```diff
@@ -425,14 +425,12 @@ class Player extends GameObject {
         let discardPile = this.getSourceList(deck + ' discard pile');
         this.game.addMessage('{0}\'s {1} deck has run out of cards, so they lose 5 honor', this, deck);
         GameActions.loseHonor({ amount: 5 }).resolve(this, this.game.getFrameworkContext());
-        this.game.queueSimpleStep(() => {
-            discardPile.each(card => this.moveCard(card, deck + ' deck'));
-            if(deck === 'dynasty') {
-                this.shuffleDynastyDeck();
-            } else {
-                this.shuffleConflictDeck();
-            }
-        });
+        discardPile.each(card => this.moveCard(card, deck + ' deck'));
```

Comment:
removing the simple step here messes up timing - the game will shuffle the deck before the player loses any honor